samples/python-v2-azurefunctions/function_app.py

```diff
@@ -0,0 +1,125 @@
+# import logging
+# import json
+# import azure.functions as func
+
+
+# def main(payload,
+#          order: func.Out[str]) -> None:
+#     logging.info(
+#         'Python function processed a CreateNewOrder request from the Dapr Runtime.')
+#     payload_json = json.loads(payload)
+#     logging.info(payload_json["data"])
+#     order.set(json.dumps({"value": payload_json["data"]}))
+
+
+import json
+import azure.functions as func
+import logging
+import json
+import azure.functions as func
+import logging
+from dapr_blueprint_app import app
+
+dapp = func.DaprFunctionApp()
+dapp.register_blueprint(app)
+
+# Dapr state output binding with http trigger
+@dapp.function_name(name="HttpTriggerFunc")
```

Comment:
Shall we split these files into multiple ones (based on the scenarios) which depicts a more real-life example.